react/src/SearchVariants/components/SearchVariantsPage.tsx

```diff
@@ -152,6 +245,37 @@ const SearchVariantsPage: React.FC<SearchVariantsPageProps> = () => {
                                 disabled={disableControls}
                             />
                         </Grid>
+                        {process.env.NODE_ENV === "development" && (
```

Comment:
This is cool and gives a good baseline for thinking about how this is all going to work once the pieces are hooked up. A few ideas from purely UX perspective:
1. can we see how this looks in a (big) modal? It might make more efficient use of real estate to put the values in columns.
2. can you sort the column names alphabetically?
3. select all / unselect all buttons
4. all columns selected by default
5. remove the chips and replace with some kind of indicator text, like "22 out of 46 columns selected" with a button next to it that says something like "add/remove columns" that pulls up the modal.